src/components/MetadataForm/__mocks__/MetadataForm.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+const MetadataForm = jest.fn().mockImplementation(() => <div data-testid="metadata-form" />);
```

Comment:
Repetitive way of mocking. kun je een helper maken.

Reply:
Fixed